.gitignore

```diff
@@ -210,6 +210,7 @@ node_modules/
 /ajax/libs/slim.js/2.[0-5].*/
 /ajax/libs/europa/4.0.0-alpha*/
 /ajax/libs/inferno-compat/*-alpha.*/
+/ajax/libs/email-autocomplete/v0.1.0
```

Comment:
@prathap0611 Thanks! It's almost there! Let's modify this to `0.1.0`. And it would be good to use another commit to do the .gitignore part :)